release/scripts/preparation_before_release.sh

```diff
@@ -22,9 +22,9 @@ set -e
 
 LOCAL_SVN_DIR=local_svn_dir
 ROOT_SVN_URL=https://dist.apache.org/repos/dist/
-DEV_REPO=dev
-RELEASE_REPO=release
-BEAM_REPO=hudi
+DEV_REPO=dev/incubator
```

Comment:
we already have a scripts directory.. can we just have these scripts there? make it `scripts/release` not to clutter the top level with too many folders.

Reply:
cc @bvaradar

Reply:
@vinothchandar : Assuming you meant the top level "tools" directory. I will go ahead and rename to  scripts and add a release folder underneath